theme/ckeditor5-ui/components/toolbar/blocktoolbar.css

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-color-block-toolbar-button: var(--ck-color-text);
+	--ck-block-toolbar-button-size: var(--ck-font-size-normal);
+}
+
+.ck.ck-block-toolbar-button {
```

Comment:
Why not `.ck-block-button`? IIRC it's `BlockButtonView` https://github.com/ckeditor/ckeditor5-ui/pull/392/files#diff-25f0cf1953c3fa884ca284a6c91b1f22R24.

Reply:
Also, where does the [`BlockButtonView`](https://github.com/ckeditor/ckeditor5-ui/pull/392/files#diff-5c933689b1a22af6c577f3f83c5a82b3R96) live in DOM? Which element is its parent? (it could have an impact on the class name)

Reply:
Button is directly in a body collection. cc @dkonopka.

Reply:
Then it makes sense, I think.